components/TimeSeriesGroup.client.vue

```diff
@@ -0,0 +1,150 @@
+          label="New per day"
+        />
+      </div>
+      <CAccordionHeader class="border-top" @click="$emit('toggleOpen')">
+        <span aria-describedby="labelDescriptor">{{ activeSeriesMetadata?.label }}</span>
+        <span id="labelDescriptor" class="visually-hidden">{{ activeSeriesMetadata?.description }}</span>
+        <CTooltip
+          v-if="activeSeriesMetadata?.description"
+          :content="activeSeriesMetadata.description"
+          placement="top"
+        >
+          <template #toggler="{ togglerId, on }">
+            <CIconSvg class="icon smaller-icon opacity-50 ms-2">
+              <img src="~/assets/icons/circleQuestion.svg" :aria-describedby="togglerId" v-on="on">
+            </CIconSvg>
+          </template>
+        </CTooltip>
+      </CAccordionHeader>
+      <CAccordionBody>
+        <TimeSeries
+          v-for="(seriesId, seriesRole, seriesIndex) in seriesGroup.time_series"
+          v-show="seriesId === activeTimeSeriesId"
+          :key="seriesId"
+          :series-role="seriesRole as string"
+          :series-id="seriesId"
+          :hide-tooltips="props.hideTooltips"
+          :series-index="seriesIndex"
+          :group-index="props.groupIndex"
+          :y-units="yUnits"
+          :chart-height="open ? chartHeightPx : minChartHeightPx"
+          @mousemove="onMove(seriesId)"
+          @touchmove="onMove(seriesId)"
+          @touchstart="onMove(seriesId)"
+          @mouseleave="$emit('hideAllTooltips')"
+          @mouseover="$emit('showAllTooltips')"
+          @chart-created="(seriesId, chart) => $emit('chartCreated', seriesId, chart)"
+          @chart-destroyed="(seriesId) => $emit('chartDestroyed', seriesId)"
+        />
+      </CAccordionBody>
+    </CAccordionItem>
+  </CAccordion>
+</template>
+
+<script lang="ts" setup>
+import { CIconSvg } from "@coreui/icons-vue";
+import type { DisplayInfo, TimeSeriesGroup } from "~/types/apiResponseTypes";
+
+const props = defineProps<{
+  seriesGroup: TimeSeriesGroup
+  groupIndex: number
+  open: boolean
+  hideTooltips: boolean
+  chartHeightPx: number
+  minChartHeightPx: number
+}>();
+
+const emit = defineEmits<{
+  hideAllTooltips: []
+  showAllTooltips: []
+  syncTooltipsAndCrosshairs: [seriesId: string]
+  toggleOpen: []
+  chartCreated: [seriesId: string, chart: Highcharts.Chart]
+  chartDestroyed: [seriesId: string]
+}>();
+
+const appStore = useAppStore();
+
+const accordionStyle = {
+  "--cui-accordion-btn-focus-box-shadow": "none",
+  "--cui-accordion-bg": "rgba(255, 255, 255, 0.7)",
+};
+
+// Since we are using a switch we assume there are only two roles.
+const timeSeriesGroupRoles = Object.keys(props.seriesGroup.time_series).slice(0, 2); // ["total", "daily"]
+const isDaily = ref(false);
+const activeTimeSeriesId = computed(() => {
+  const activeTimeSeriesRole = timeSeriesGroupRoles[Number(isDaily.value)]; // total or daily
+  return props.seriesGroup.time_series[activeTimeSeriesRole];
+});
```

Comment:
Since we're effectively hardcoding to "total" and "daily" roles here, I think just using the strings "total" and "daily" would be more readable and not any more presumptuous that assuming that the first two keys in the group correspond to those roles, in an assumed order.